src/System.Reflection.Metadata/ref/System.Reflection.Metadata.cs

```diff
@@ -99,6 +99,7 @@ public partial struct AssemblyDefinition
         public System.Reflection.Metadata.StringHandle Name { get { throw null; } }
         public System.Reflection.Metadata.BlobHandle PublicKey { get { throw null; } }
         public System.Version Version { get { throw null; } }
+        public System.Reflection.AssemblyName AssemblyName { get { throw null; } }
```

Comment:
This should instead be a method and not a property as that was the approved API. 

```cs
public System.Reflection.AssemblyName GetAssemblyName()
```